### ISSUE TITLE
argparse -"-p", "--port",  TypeError: an integer is required (got type str)

### ISSUE THREAD

**Issue by QGB** [cs01/pyxtermjs#19]
```
>python3 -m pyxtermjs -p 443                                                                                                        
serving on http://127.0.0.1:443
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/klk/.local/lib/python3.6/site-packages/pyxtermjs/__main__.py", line 5, in <module>
    exit(main())
  File "/home/klk/.local/lib/python3.6/site-packages/pyxtermjs/app.py", line 123, in main
    socketio.run(app, debug=args.debug, port=args.port, host=args.host)
  File "/home/klk/.local/lib/python3.6/site-packages/flask_socketio/__init__.py", line 584, in run
    self.wsgi_server.serve_forever()
  File "/usr/local/lib/python3.6/dist-packages/gevent/baseserver.py", line 360, in serve_forever
    self.start()
  File "/usr/local/lib/python3.6/dist-packages/gevent/baseserver.py", line 304, in start
    self.init_socket()
  File "/usr/local/lib/python3.6/dist-packages/gevent/pywsgi.py", line 1451, in init_socket
    StreamServer.init_socket(self)
  File "/usr/local/lib/python3.6/dist-packages/gevent/server.py", line 127, in init_socket
    self.socket = self.get_listener(self.address, self.backlog, self.family)
  File "/usr/local/lib/python3.6/dist-packages/gevent/server.py", line 138, in get_listener
    return _tcp_listener(address, backlog=backlog, reuse_addr=cls.reuse_addr, family=family)
  File "/usr/local/lib/python3.6/dist-packages/gevent/server.py", line 228, in _tcp_listener
    sock.bind(address)
TypeError: an integer is required (got type str)

parser.add_argument("-p", "--port", default=5000, help="port to run server on")

should add `type=int`
```

**Comment by cs01** [comment]
This was fixed in v0.5.0.1

**Comment by QGB** [comment]
too slow to fix
